modules/gatk/realignertargetcreator/main.nf

```diff
@@ -0,0 +1,43 @@
+process GATK_REALIGNERTARGETCREATOR {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::gatk=3.5" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gatk:3.5--hdfd78af_11':
+        'quay.io/biocontainers/gatk:3.5--hdfd78af_11' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+    tuple path(fasta), path(fasta_fai), path(fasta_dict)
+    path(known_vcf)
+
+    output:
+    tuple val(meta), path("*.intervals"), emit: intervals
+    path "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def known = known_vcf ? "-known ${known_vcf}" : ""
+    if ("$bam" == "${prefix}.bam") error "Input and output names are the same, set prefix in module configuration to disambiguate!"
+
```

Comment:
```
def avail_mem = 3
    if (!task.memory) {
        log.info '[GATK GenotypeGVCFs] Available memory not known - defaulting to 3GB. Specify process memory requirements to change this.'
    } else {
        avail_mem = task.memory.giga
    }
    """
    gatk --java-options "-Xmx${avail_mem}g" GenotypeGVCFs \\
    ```
    
    we have been using this for all gatk modules, something about java nad making sure it doesn't fail or so. NOt sure if also applicable for gatk3